### PR TITLE
Added selenium tests bootstrapping

### PR DESCRIPTION
**Pull request by santhoshb-msft**
Added selenium tests bootstrapping

### CODE REVIEW

**Comment by santhoshb-msft** [comment]
This is a draft PR to show whats coming. Will build a new PR on UI tests once the tests folder structure PR is merged

**Review by dstarr — CHANGES_REQUESTED** [review]
I know you knwo this, but after getting the tests to run, please:

Create 2 projects for these tests
- AdminSite.Test.Ui
- CustomerSite.Test.Ui

The Services project has no UI

Really like this first stab at UI testing!

File: src/Services.Test.Ui/AdminApplicationShould.cs [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+
+        //Act
+        driver.FindElement(By.Id("i0116")).Clear();
+        driver.FindElement(By.Id("i0116")).SendKeys(loginUserName);
+        driver.FindElement(By.Id("i0116")).SendKeys(Keys.Enter);
+        Thread.Sleep(2000);
+        driver.FindElement(By.Id("i0118")).SendKeys(password);
+        driver.FindElement(By.Id("i0118")).SendKeys(Keys.Enter);
+        Thread.Sleep(2000);
+        driver.FindElement(By.Id("idSIButton9")).SendKeys(Keys.Enter);
```
[the remark this answers is not in the document]
Comment (santhoshb-msft) [reply src/Services.Test.Ui/AdminApplicationShould.cs:89]: image
 Works for me @dstarr

[the remark this answers is not in the document]
Comment (santhoshb-msft) [reply src/Services.Test.Ui/AdminApplicationShould.cs:89]: @dstarr I believe we resolve this in our offline conversation.

**Pull request closed without merging**